app/services/ExpressionGenerator.coffee

```diff
@@ -0,0 +1,37 @@
+class ExpressionGenerator
+
+	randInclusive: (min, max) ->
+		Math.floor(Math.random() * (max - min + 1)) + min;
+
+	genRandomDigit: (min, max) ->
+		@randInclusive(min, max).toString()
+
+	genRandomOperator: ->
+		switch @randInclusive 0, 2
+			when 0 then "+"
+			when 1 then "-"
+			when 2 then "*"
+			else "?"
```

Comment:
Instead of a `switch` statement, you could create a string or array containing the operators, then `@randInclusive 0, 2` returns the index to that array.

Crappy github-pseudocode below
```
ops = ("+×-").toCharArray()
ops[@randInclusive 0, 2]
```